[2020-07-04T08:10:31.981] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:10:36.814] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    --------------------
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:10:39.031] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
From previous event:
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\collection.js:160:12)
From previous event:
    at Child.fetchAll (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:878:10)
    at Function.fetchAll (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\bookshelf.js:287:29)
    at Object.out.List (C:\Users\BIM02\Desktop\ServerKT\restful\v0\deps.js:10:25)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-compose\index.js:42:32)
    at parseBody.then.body (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-bodyparser\index.js:74:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:10:44.018] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
From previous event:
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\collection.js:160:12)
From previous event:
    at Child.fetchAll (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:878:10)
    at Function.fetchAll (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\bookshelf.js:287:29)
    at Object.out.List (C:\Users\BIM02\Desktop\ServerKT\restful\v0\deps.js:10:25)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-compose\index.js:42:32)
    at parseBody.then.body (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-bodyparser\index.js:74:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:10:46.223] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
From previous event:
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:204:8)
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-compose\index.js:42:32)
    at parseBody.then.body (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-bodyparser\index.js:74:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:15:12.944] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
From previous event:
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:204:8)
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-compose\index.js:42:32)
    at parseBody.then.body (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-bodyparser\index.js:74:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:30:56.844] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
From previous event:
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:204:8)
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-compose\index.js:42:32)
    at parseBody.then.body (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-bodyparser\index.js:74:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-04T08:34:11.269] [ERROR] app - { Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:273:13)
    at Socket.emit (events.js:182:13)
    at Socket._onTimeout (net.js:449:8)
    at ontimeout (timers.js:425:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:116:18)
    at Promise (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:69:18)
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:291:39)
From previous event:
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:204:8)
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
From previous event:
    at Sync.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:702:19)
    at Object.out.Post (C:\Users\BIM02\Desktop\ServerKT\restful\v0\sessions.js:37:8)
    at RestfulAPIMethods.(anonymous function) (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:32:60)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at route.use (C:\Users\BIM02\Desktop\ServerKT\routes\restful.js:97:19)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:346:16
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-compose\index.js:42:32)
    at parseBody.then.body (C:\Users\BIM02\Desktop\ServerKT\node_modules\koa-bodyparser\index.js:74:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true }
[2020-07-06T10:55:43.626] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, '18a9f577-12a6-4584-94cc-2ae805e6606b', '3123', NULL, 0, '4151') - ER_DUP_ENTRY: Duplicate entry '4151' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'4151\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'18a9f577-12a6-4584-94cc-2ae805e6606b\', \'3123\', NULL, 0, \'4151\')' }
[2020-07-06T10:56:14.170] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, 'b64b5251-c6ce-454d-82eb-d40cb80be9e8', '3123', NULL, 0, '4151') - ER_DUP_ENTRY: Duplicate entry '4151' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'4151\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'b64b5251-c6ce-454d-82eb-d40cb80be9e8\', \'3123\', NULL, 0, \'4151\')' }
[2020-07-06T10:56:38.543] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, '71c2857d-f1a1-48f7-a815-782dc040201e', '4123', '234') - ER_DUP_ENTRY: Duplicate entry '234' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'234\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'71c2857d-f1a1-48f7-a815-782dc040201e\', \'4123\', \'234\')' }
[2020-07-06T13:41:10.050] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `phone`, `user`) values (DEFAULT, 'ea97c815-fbb8-45fc-89fc-63d56e42bfab', '12345', '153321432', '234') - ER_DUP_ENTRY: Duplicate entry '234' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'234\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `phone`, `user`) values (DEFAULT, \'ea97c815-fbb8-45fc-89fc-63d56e42bfab\', \'12345\', \'153321432\', \'234\')' }
[2020-07-06T14:46:32.621] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, '393f9545-1a91-4470-ae65-68d13c77dd80', '444', NULL, 0, '32') - ER_DUP_ENTRY: Duplicate entry '444' for key 'user_name_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'444\' for key \'user_name_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'393f9545-1a91-4470-ae65-68d13c77dd80\', \'444\', NULL, 0, \'32\')' }
[2020-07-06T14:50:21.460] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, '4ef08670-dd6c-4aac-885e-39d9eeff92d5', '1231', NULL, 0, '4124123') - ER_DUP_ENTRY: Duplicate entry '4124123' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'4124123\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'4ef08670-dd6c-4aac-885e-39d9eeff92d5\', \'1231\', NULL, 0, \'4124123\')' }
[2020-07-06T14:50:25.874] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, '25ad428e-ad39-49a5-9626-e0915bbe8197', '1231', NULL, 0, '41241235') - ER_DUP_ENTRY: Duplicate entry '1231' for key 'user_name_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'1231\' for key \'user_name_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'25ad428e-ad39-49a5-9626-e0915bbe8197\', \'1231\', NULL, 0, \'41241235\')' }
[2020-07-06T14:50:53.412] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, '69d0a558-c900-4e34-9a4d-abcff1ae28f4', '3124', NULL, 0, '1231a55') - ER_DUP_ENTRY: Duplicate entry '3124' for key 'user_name_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'3124\' for key \'user_name_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'69d0a558-c900-4e34-9a4d-abcff1ae28f4\', \'3124\', NULL, 0, \'1231a55\')' }
[2020-07-06T14:51:20.491] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, 'f6a15065-9c39-4fd2-a912-800a83de83ea', '41', NULL, 0, '41231') - ER_DUP_ENTRY: Duplicate entry '41231' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'41231\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'f6a15065-9c39-4fd2-a912-800a83de83ea\', \'41\', NULL, 0, \'41231\')' }
[2020-07-06T14:51:49.078] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, '1fe4718e-28f8-4051-9499-b3df4ff1e87b', '41', NULL, 0, '41231') - ER_DUP_ENTRY: Duplicate entry '41231' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'41231\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'1fe4718e-28f8-4051-9499-b3df4ff1e87b\', \'41\', NULL, 0, \'41231\')' }
[2020-07-06T14:52:01.369] [ERROR] app - { Error: insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, 'f46f8e74-ab6a-43dd-a923-3e473a021658', '41', NULL, 0, '41231') - ER_DUP_ENTRY: Duplicate entry '41231' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'41231\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`avatar`, `created_at`, `created_by`, `id`, `name`, `phone`, `state`, `user`) values (NULL, NULL, DEFAULT, \'f46f8e74-ab6a-43dd-a923-3e473a021658\', \'41\', NULL, 0, \'41231\')' }
[2020-07-06T14:54:31.987] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, 'b13ca8c8-5eaa-4fbf-8877-c26d539bf80b', '4123', '4123') - ER_DUP_ENTRY: Duplicate entry '4123' for key 'user_name_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'4123\' for key \'user_name_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'b13ca8c8-5eaa-4fbf-8877-c26d539bf80b\', \'4123\', \'4123\')' }
[2020-07-06T15:51:46.923] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, '4db25d9e-97d8-4438-96f5-dbd3eefbf50c', '32', '123a') - ER_DUP_ENTRY: Duplicate entry '123a' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'123a\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'4db25d9e-97d8-4438-96f5-dbd3eefbf50c\', \'32\', \'123a\')' }
[2020-07-06T15:52:43.842] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, '5926b64c-d66f-4f28-ace2-4ce3f2a9c64d', '32', '123a') - ER_DUP_ENTRY: Duplicate entry '123a' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'123a\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'5926b64c-d66f-4f28-ace2-4ce3f2a9c64d\', \'32\', \'123a\')' }
[2020-07-06T15:53:04.455] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, '20fe2a04-4769-4f1e-b63e-4fff1510dfc1', '32', '123a') - ER_DUP_ENTRY: Duplicate entry '123a' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'123a\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'20fe2a04-4769-4f1e-b63e-4fff1510dfc1\', \'32\', \'123a\')' }
[2020-07-06T15:53:46.832] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, '774e61f1-11b8-49cc-a20d-af6f02e67c59', '43', '123a') - ER_DUP_ENTRY: Duplicate entry '123a' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'123a\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'774e61f1-11b8-49cc-a20d-af6f02e67c59\', \'43\', \'123a\')' }
[2020-07-06T15:58:25.245] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, '5eee96c3-6ad8-4568-af6b-836f41fb3a3c', '432', '123a') - ER_DUP_ENTRY: Duplicate entry '123a' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'123a\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'5eee96c3-6ad8-4568-af6b-836f41fb3a3c\', \'432\', \'123a\')' }
[2020-07-06T15:59:31.152] [ERROR] app - { Error: insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, 'ec1b4b10-e1a6-4f04-9fee-804612a742fc', '43', '123a') - ER_DUP_ENTRY: Duplicate entry '123a' for key 'user_user_unique'
    at Query.Sequence._packetToError (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
    at Protocol._enqueue (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Connection.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\mysql\lib\Connection.js:198:25)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:135:18
    at new Promise (<anonymous>)
    at Client_MySQL._query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\dialects\mysql\index.js:129:12)
    at Client_MySQL.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\client.js:169:17)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:321:24
    at new Promise (<anonymous>)
    at Client_MySQL.trxClient.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:316:12)
    at Runner.query (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:134:36)
    at C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:40:23
    at client.acquireConnection.catch.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\runner.js:260:24)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1160:42)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1159:14)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at Child.<anonymous> (C:\Users\BIM02\Desktop\ServerKT\node_modules\bookshelf\lib\model.js:1060:10)
From previous event:
    at B.transaction.trx (C:\Users\BIM02\Desktop\ServerKT\restful\v0\users.js:35:29)
    at init.then.then (C:\Users\BIM02\Desktop\ServerKT\node_modules\knex\lib\transaction.js:184:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'123a\' for key \'user_user_unique\'',
  sqlState: '23000',
  index: 0,
  sql:
   'insert into `user` (`created_by`, `id`, `name`, `user`) values (DEFAULT, \'ec1b4b10-e1a6-4f04-9fee-804612a742fc\', \'43\', \'123a\')' }
